__test__/cypress/plugins/index.js

```diff
@@ -1,7 +1,6 @@
 // plugins run in the cypress Node process, not the browser. Define tasks
 // operations like seeding the database that can't be run from the browser.
 // See: https://on.cypress.io/plugins-guide
-require("dotenv").load();
```

Comment:
I scratched my head over this for a while. Is `.env` used by webpack in ways that I don't understand? This line was causing webpack to fail to build in CI where `.env` does not exist. Why doesn't the Heroku Dockerfile build fail in the same way?